### ISSUE TITLE
no Mouse Toggle Service Switch in Accessibility section

### ISSUE THREAD

**Issue by Mikka123T** [virresh/matvt#30]
Hi Virresh, I have installed apk on TOSHIBA 58UA2063DG (Android TV 9). I have enabled Display Over other  Apps. However there is no Mouse Toggle Service Switch in Accessibility section of Device prefs so I cannot configure the keycode. I have also tried to install other version (activated with Info key) and no success neither. Can you  help with this? Thanks

**Comment by virresh** [comment]
Hi @Mikka123T ,
ADB commands for these can be found here - https://github.com/virresh/matvt/issues/18#issuecomment-884013775

It may not work on some manufacturer devices that have removed accessibility APIs, not sure if TOSHIBA 58UA2063DG is one of them. Please try the commands and let us know if they work
